src/api/public/apidocs-new/paths/staging_project_name_staged_requests.yaml

```diff
@@ -0,0 +1,75 @@
+delete:
+  summary: Delete requests from any of the staging projects.
+  description: Delete the given requests from any of the staging projects they were staged in.
+  security:
+    - basic_authentication: []
+  parameters:
+    - $ref: '../components/parameters/project_name.yaml'
+  requestBody:
+    description: List of requests to be removed from the staging projects.
```

Comment:
I'd rather use `deleted` instead of `removed` for consistency with the rest of the text of this endpoint.
And I'd actually also omit the `from the staging projects` here, it has already been said in the endpoint description, here instead it's only a matter of defining the `requestBody` data.